lua/nvim-navbuddy/actions.lua

```diff
@@ -311,11 +360,14 @@ local function swap_nodes(for_buf, nodeA, nodeB)
 		return
 	end
 
-	local nodeA_text = vim.api.nvim_buf_get_lines(for_buf, nodeA.scope["start"].line-1, nodeA.scope["end"].line-1+1, false)
-	local mid_text = vim.api.nvim_buf_get_lines(for_buf, nodeA.scope["end"].line-1+1, nodeB.scope["start"].line-1, false)
-	local nodeB_text = vim.api.nvim_buf_get_lines(for_buf, nodeB.scope["start"].line-1, nodeB.scope["end"].line-1+1, false)
+	local nodeA_text =
+		api.nvim_buf_get_lines(for_buf, nodeA.scope["start"].line - 1, nodeA.scope["end"].line - 1 + 1, false)
+	local mid_text =
+		api.nvim_buf_get_lines(for_buf, nodeA.scope["end"].line - 1 + 1, nodeB.scope["start"].line - 1, false)
+	local nodeB_text =
+		api.nvim_buf_get_lines(for_buf, nodeB.scope["start"].line - 1, nodeB.scope["end"].line - 1 + 1, false)
```

Comment:
Don't make style changes as part of same pr please.

Reply:
yep sry it's the auto format... ^^